server/ServerApp.cpp

```diff
@@ -1799,6 +1808,18 @@ bool ServerApp::IsHostless() const
 const boost::circular_buffer<ChatHistoryEntity>& ServerApp::GetChatHistory() const
 { return m_chat_history; }
 
+std::vector<PlayerSaveGameData> ServerApp::GetPlayerSaveGameData() const {
+    std::vector<PlayerSaveGameData> player_save_game_data;
+    for (const auto& m_save_data : m_turn_sequence) {
+        DebugLogger() << "Empire " << m_save_data.first << " ready " << m_save_data.second.first
+            << " save_game_data " << m_save_data.second.second.get();
```

Comment:
indent to match first `<<` on preceeding line

Reply:
Fixed.